extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -539,6 +538,13 @@ public static class Jwks {
         @ConfigItem
         public Optional<Duration> cleanUpTimerInterval = Optional.empty();
 
+        /**
+         * In case there is no kid or x5t specified in the JOSE header and no key could be determined, check all available keys. Defaults to: false.
```

Comment:
```suggestion
         * In case there is no 'kid' or 'x5t' specified in the JOSE header and no key could be determined, check all available keys matching a token algorithm ('alg') header value.```

Reply:
The generated doc will show what a default value is

Reply:
Updated